syntax = "proto3";

import "super.proto";

service Booking {
    rpc GetBookingsByUser(UserID) returns (stream BookingData) {}
    rpc GetBookings(Empty) returns (stream BookingData) {}
    rpc AddBookingByUser(BookingData) returns (BookingData) {}
}

message UserID {
    string userid = 1;
}

message BookingData {
    string userid = 1;
    TimeShow date = 2;
}